### PR TITLE
[OraTV] update extractor

### PR DESCRIPTION
**Pull request by CeruleanSky**
"current" is now "video"
"hls_stream" is now hls_stream without quotes
video_id is now id
duration for current video is not present(for other videos it is)

### CODE REVIEW

**Comment by CeruleanSky** [comment]
This fixes the broken Ora TV extractor due to changes in the variable names in the webpage.  It removes the duration test , as that information is no longer present for the current video that is being requested to be downloaded.  For some reason duration is still listed for the suggested videos though, so it may return, but in the meantime it is better not to test for something that isn't there.

File: youtube_dl/extractor/ora.py [hunk 1]
```diff
@@ -30,9 +29,9 @@ def _real_extract(self, url):
         webpage = self._download_webpage(url, display_id)
 
         video_data = self._search_regex(
-            r'"current"\s*:\s*({[^}]+?})', webpage, 'current video')
+            r'"video"\s*:\s*({[^}]+?})', webpage, 'current video')
         m3u8_url = self._search_regex(
-            r'"hls_stream"\s*:\s*"([^"]+)', video_data, 'm3u8 url', None)
+            r'hls_stream\s*:\s*"([^"]+)', video_data, 'm3u8 url', None)
```
Comment (dstftw) [inline youtube_dl/extractor/ora.py]: Both quoted and quoteless should be tried. Looking at their JSON they can switch back any moment.

**Comment by CeruleanSky** [comment]
Modified regex to take into account that hls_stream may be put in quotes again in the future.

**Comment by dstftw** [comment]
Cherry-picked in 1f78ed189aab9b7ea17594ac1f33662d37433455, thanks.

**Pull request closed without merging**